File: OCH_Tao/ABC/B/241.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const A = input[1].split(" ").map(x => parseInt(x, 10));
const B = input[2].split(" ").map(x => parseInt(x, 10));
const C = A.reduce((x, y) => x.set(y, (x.get(y) ?? 0) + 1), new Map());
for (const i of B) {
  if (!C.has(i) || C.get(i) === 0) {
    console.log("No");
    process.exit();
  } else {
    C.set(i, C.get(i) - 1);
  }
}
console.log("Yes");

File: OCH_Tao/ABC/B/241.py
from collections import Counter
N,M = map(int,input().split())
A = Counter(list(map(int,input().split())))
B = list(map(int,input().split()))
for i in B:
  if A[i]==0:
    print("No")
    break
  else:
    A[i]-=1
else:
  print("Yes")